ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/TransactionsReader.scala

```diff
@@ -318,14 +318,16 @@ private[appendonlydao] final class TransactionsReader(
         )
       }
 
+    val transactionLogUpdatesSource = groupContiguous(eventsSource)(by = _.transactionId)
+      .map { v =>
+        val tx = toTransaction(v)
+        (tx.offset, tx.events.last.eventSequentialId) -> tx
+      }
+      .mapMaterializedValue(_ => NotUsed)
+
     InstrumentedSource
       .bufferedSource(
-        original = groupContiguous(eventsSource)(by = _.transactionId)
```

Comment:
Unrelated to this PR: aestethics refactoring

Reply:
As long as you preserve `groupContiguous(eventsSource)(by = _.transactionId)`. I don't know who wrote but the original author clearly has a great taste. :trollface:

Reply:
Original author :trollface:

Reply:
>As long as you preserve groupContiguous(eventsSource)(by = _.transactionId). I don't know who wrote but the original author clearly has a great taste. 

@stefanobaghino-da Or actually, wdym? Maybe I'm missing something

Reply:
Just kidding. 😉 LGTM.